app/jobs/scheduled/clean_dismissed_topic_users.rb

```diff
@@ -24,7 +24,7 @@ def delete_overdue_dismissals!
                     WHEN COALESCE(user_options.new_topic_duration_minutes, :default_duration) = :always THEN users.created_at
                     WHEN COALESCE(user_options.new_topic_duration_minutes, :default_duration) = :last_visit THEN COALESCE(users.previous_visit_at,users.created_at)
                     ELSE (:now::timestamp - INTERVAL '1 MINUTE' * COALESCE(user_options.new_topic_duration_minutes, :default_duration))
-                 END, user_stats.new_since, :min_date)
+                 END, users.created_at, :min_date)
```

Comment:
I could keep here new_since as a column still existing and we will potentially get rid of that later. I wanted to already switch to use `created_at`. The reason is that after registration, the user should have an empty "new" tab. Only topics created after could be `new`